n2y/converter.py

```diff
@@ -163,18 +164,48 @@ def __init__(self, block):
         for key, value in block.items():
             self.__dict__[key] = value
 
+    def handler(self, annotation, target):
+        handler = {
+            'bold': [Strong([target]), Strong(target)],
+            'italic': [Emph([target]), Emph(target)],
+            'strikethrough': [Strikeout([target]), Strikeout(target)]
+        }
+        return handler[annotation]
+
+    def strip(self, target):
```

Comment:
I'll leave it to @xaker00 to review this algorithm in detail, but it looks nice at first glance!